libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -224,7 +222,10 @@ namespace
 			json_obj["@timestamp"] = time_ms;
 
 			insert_or_parse_as_bin(json_obj, "hostname", host_name);
-			json_obj["pid"] = std::to_string(pid);
+
+			pid_t pid = getpid();
+			json_obj["pid"] = pid;
```

Comment:
Could we do like line 372 here?
```suggestion
			json_obj["pid"] = getpid();
```

Reply:
No, because `pid` is used on line 407.

Reply:
Ah, the trouble with commit-level viewing. Got it.